packages/api/src/client/ws/index.ts

```diff
@@ -0,0 +1,26 @@
+// @ts-ignore
+import SocketRequestClient from 'socket-request-client'
+import { BlueAsset } from '@blueserver/types';
+
+
+export class WSApiClient {
+  // @ts-ignore
+  #client;
+
+  constructor() {
+    this.#init() 
+  }
+
+  async #init() {
+    this.#client = new SocketRequestClient('ws://blue.leofcoin.org', 'protocol-blue')
```

Comment:
ERRORS!!!!!

Reply:
let's dismiss that for now

Reply:
Yeah, I need to fix declarations, the module itself works.